### PR TITLE
add c5.12xlarge, c5.24xlarge, c5.metal, i3en.metal

### PR DESCRIPTION
**Pull request by rekcah78**
update instances with make update-machine-types

### CODE REVIEW

File: upup/pkg/fi/cloudup/awsup/machine_types.go [hunk 1]
```diff
@@ -80,6 +80,15 @@ var MachineTypes []AWSMachineTypeInfo = []AWSMachineTypeInfo{
...
 	// BEGIN GENERATED CONTENT
 
+	// WARNING: ENIs not known for i3en.metal
+	// WARNING: IPs per ENI not known for i3en.metal
+	// WARNING: ENIs not known for c5.24xlarge
+	// WARNING: IPs per ENI not known for c5.24xlarge
+	// WARNING: ENIs not known for c5.12xlarge
+	// WARNING: IPs per ENI not known for c5.12xlarge
+	// WARNING: ENIs not known for c5.metal
+	// WARNING: IPs per ENI not known for c5.metal
```
Comment (mikesplain) [inline upup/pkg/fi/cloudup/awsup/machine_types.go]: In order to remove these warnings, please manually add the required data here: https://github.com/kubernetes/kops/blob/master/hack/machine_types/vpc_ip_resource_limit.go

Above the two sections you see links to where you should be able to get them!

Comment (mikesplain) [reply upup/pkg/fi/cloudup/awsup/machine_types.go]: Then rerun the make command to regenerate. Thanks!

Comment (rekcah78) [reply upup/pkg/fi/cloudup/awsup/machine_types.go]: thanks for that, https://docs.aws.amazon.com/en_us/AWSEC2/latest/UserGuide/using-eni.html is not updated yet, I'm trying to get this through a contact at AWS.

Comment (rekcah78) [reply upup/pkg/fi/cloudup/awsup/machine_types.go]: @mikesplain done but now i3en.metal is in the list but number of max ENI and max IP per ENI is not available for this instance type :)

**Comment by justinsb** [comment]
Thanks for filling in the ones that are known.  We can go with this as-is I believe, and then when AWS publishes the specs for i3en.metal we can fill those in!

/approve
/lgtm
/retest

**Comment by rekcah78** [comment]
@mikesplain I have to open a PR for merge in 1.13 and 1.14?

**Comment by mikesplain** [comment]
@rekcah78 Done 😄 
Feel free to `/lgtm` those cherry picks if they look good to you!

**Pull request merged**